npm-diff

```diff
@@ -56,13 +74,19 @@ for job in `jobs -p`; do wait $job || exit 1; done
 
 # diff
 
-diff \
-  --recursive \
-  --unified \
-  --exclude test \
-  --exclude Makefile \
-  $a $b \
-  | egrep -v "\"readme\"|\"_id\"|\"_from\"|\"_resolved\""
+diffCommand='diff 
+  --recursive 
+  --unified 
+  --exclude test 
+  --exclude Makefile 
+  $a $b 
+  | egrep -v "\"readme\"|\"_id\"|\"_from\"|\"_resolved\""'
+
+if [ "$color" = true ] ; then
+    diffCommand+="| colordiff"
```

Comment:
if all it takes is for people to pipe to `colordiff`, wouldn't it be easier for them to just create a shell alias? I'd imagine that if I were to use `npm-diff` with `colordiff`, I'd create an alias to, so I wouldn't have to add the `-c` option all the time.

What do you think about instead of adding an argument we make it use `colordiff` by default, if available?

Reply:
Yes, the alias would be a solution, but I was thinking in a user that doesn't know/want to create one or thing it is easier to just pass a command.

But yes, I agree with the idea to use it by default and if not available, fallback to how it works today.

I will do these changes over the weekend and let you know when it is pushed.

Thanks

Reply:
Hi @juliangruber,

Finally I got time to work again on this. 

I have implemented as you had suggested, now it will use `colordiff` by default if it is available, otherwise it will output the normal no color text.

Please let me know if you need any other change or if you have any question/comment.

Thank you !